doc/man3/OPENSSL_load_u16_le.pod

```diff
@@ -0,0 +1,83 @@
+=pod
+
+=head1 NAME
+
+OPENSSL_load_u16_le, OPENSSL_load_u16_be, OPENSSL_load_u32_le,
+OPENSSL_load_u32_be, OPENSSL_load_u64_le, OPENSSL_load_u64_be,
+OPENSSL_store_u16_le, OPENSSL_store_u16_be,
+OPENSSL_store_u32_le, OPENSSL_store_u32_be,
+OPENSSL_store_u64_le, OPENSSL_store_u64_be -
+Read and write unsigned 16, 32 and 64-bit integers in a specific byte order
+
+=head1 SYNOPSIS
+
+    #include <openssl/byteorder.h>
+
+    static ossl_inline unsigned char *OPENSSL_store_u16_le(
+        unsigned char *out, uint16_t val);
+    static ossl_inline unsigned char *OPENSSL_store_u16_be(
+        unsigned char *out, uint16_t val);
+    static ossl_inline unsigned char *OPENSSL_store_u32_le(
+        unsigned char *out, uint32_t val);
+    static ossl_inline unsigned char *OPENSSL_store_u32_be(
+        unsigned char *out, uint32_t val);
+    static ossl_inline unsigned char *OPENSSL_store_u64_le(
+        unsigned char *out, uint64_t val);
+    static ossl_inline unsigned char *OPENSSL_store_u64_be(
+        unsigned char *out, uint64_t val);
+    static ossl_inline const unsigned char *OPENSSL_load_u16_le(
+        uint16_t *val, const unsigned char *in);
+    static ossl_inline const unsigned char *OPENSSL_load_u16_be(
+        uint16_t *val, const unsigned char *in);
+    static ossl_inline const unsigned char *OPENSSL_load_u32_le(
+        uint32_t *val, const unsigned char *in);
+    static ossl_inline const unsigned char *OPENSSL_load_u32_be(
+        uint32_t *val, const unsigned char *in);
+    static ossl_inline const unsigned char *OPENSSL_load_u64_le(
+        uint64_t *val, const unsigned char *in);
+    static ossl_inline const unsigned char *OPENSSL_load_u64_be(
+        uint64_t *val, const unsigned char *in);
+
+=head1 DESCRIPTION
+
+These functions read and write 16, 32 and 64 bit unsigned integers in a
+specified byte order.
+The I<_be> functions use big-endian byte order, while the I<_le> functions
+use little-endian byte order.
+They're implemented directly in the header file, and declared static.
+When the compiler supports inline functions, they're also declared inline.
+An optimising compiler will often convert these to just one or two machine
+instructions: a load or store with a possible byte swap.
+
+The I<load> functions write the decoded integer value at the address pointed to by
+B<val>, which must be a valid (possibly suitably aligned) address of an object of
+the appropriate type.
+The I<store> functions write the encoding of B<val> at the address pointed to by B<out>.
```

Comment:
Please use `C<_be> C<_le> C<load> C<store>` functions. And `I<val> and I<out>`.

Reply:
Maybe we should document those things in a style guide. As far as I know, we currently don't have anything like that for documentation.

Reply:
We try to follow man-pages(7) manpage in the manual page formatting.

Reply:
Done.